### PR TITLE
add threat model details

### PR DESCRIPTION
**Pull request by JimBugwadia**
Signed-off-by: Jim Bugwadia <jim@nirmata.com>

See: https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md

Added details for each threat.

There is some redundancy but seems like the best tradeoff to be complete.

### CODE REVIEW

File: content/en/docs/security/_index.md [hunk 1]
```diff
@@ -158,4 +155,167 @@ At a minimum, the [Pod Security Standards](/policies/pod-security/) and [best pr
...
+
+     By default, Kyverno generates a CA and X.509 certificates for the webhook registration. As custom CA and certificates can be used as discussed in the [installation guide](/docs/installation/#option-2-use-your-own-ca-signed-certificate).  
+
+### [Threat ID 10 - Abusing a mutation rule to create a privileged container](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#threat-id-10---abusing-a-mutation-rule-to-create-a-privileged-container)
+
+**Mitigation**
+
+* [Mitigation ID 6 - All rules are reviewed and tested](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#Mitigation-id-6---all-rules-are-reviewed-and-tested)
+
+     Kyverno rules are Kubernetes resources written in YAML and managed by a declarative OpenAPIv3 schema. This makes it easy understand  policy and to apply `policy as code` practices, such as code reviews, to Kyverno policies.
```
Comment (chipzoller) [inline content/en/docs/security/_index.md:250]: "This makes it easy to understand policy and to apply policy as code practices, such as code reviews, to Kyverno policies." (insert "to", delete space between "understand" and "policy", remove code formatting from "policy as code" phrase)

File: content/en/docs/security/_index.md [hunk 2]
```diff
@@ -158,4 +155,167 @@ At a minimum, the [Pod Security Standards](/policies/pod-security/) and [best pr
...
+
+     Kyverno does not exempt any namespaces by default. It allows configuration of exempt namespaces via a [ConfigMap](https://kyverno.io/docs/installation/#configmap-flags).
+
+### [Threat ID 12 - Block rule can be bypassed due to missing match (e.g. missing initcontainers)](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#threat-id-12---block-rule-can-be-bypassed-due-to-missing-match-eg-missing-initcontainers)
+
+**Mitigation**
+
+* [Mitigation ID 6 - All rules are reviewed and tested](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#Mitigation-id-6---all-rules-are-reviewed-and-tested)
+
+     Kyverno rules are Kubernetes resources written in YAML and managed by a declarative OpenAPIv3 schema. This makes it easy understand  policy and to apply `policy as code` practices, such as code reviews, to Kyverno policies. The [Kyverno CLI](/docs/kyverno-cli/) provides a `test` command for executing unit tests as part of the `policy as code` pipeline.
```
Comment (chipzoller) [inline content/en/docs/security/_index.md:268]: Same changes as above.

File: content/en/docs/security/_index.md [hunk 3]
```diff
@@ -158,4 +155,167 @@ At a minimum, the [Pod Security Standards](/policies/pod-security/) and [best pr
...
+
+### [Threat ID 13 - Attacker exploits bad string matching on a blocklist to bypass rules](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#threat-id-13---attacker-exploits-bad-string-matching-on-a-blocklist-to-bypass-rules)
+
+**Mitigation**
+
+* [Mitigation ID 6 - All rules are reviewed and tested](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#Mitigation-id-6---all-rules-are-reviewed-and-tested)
+
+     Kyverno rules are Kubernetes resources written in YAML and managed by a declarative OpenAPIv3 schema. This makes it easy understand  policy and to apply `policy as code` practices, such as code reviews, to Kyverno policies. The [Kyverno CLI](/docs/kyverno-cli/) provides a `test` command for executing unit tests as part of the `policy as code` pipeline.
```
Comment (chipzoller) [inline content/en/docs/security/_index.md:276]: Ditto above

File: content/en/docs/security/_index.md [hunk 4]
```diff
@@ -158,4 +155,167 @@ At a minimum, the [Pod Security Standards](/policies/pod-security/) and [best pr
...
+
+### [Threat ID 14 - Attacker uses new/old features of the Kubernetes API which have no rules](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#threat-id-14---attacker-uses-newold-features-of-the-kubernetes-api-which-have-no-rules)
+
+**Mitigation**
+
+* [Mitigation ID 6 - All rules are reviewed and tested](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#Mitigation-id-6---all-rules-are-reviewed-and-tested)
+
+     Kyverno rules are Kubernetes resources written in YAML and managed by a declarative OpenAPIv3 schema. This makes it easy understand  policy and to apply `policy as code` practices, such as code reviews, to Kyverno policies. The [Kyverno CLI](/docs/kyverno-cli/) provides a `test` command for executing unit tests as part of the `policy as code` pipeline.
```
Comment (chipzoller) [inline content/en/docs/security/_index.md:284]: Ditto above.

**Review by chipzoller — DISMISSED** [review]
Looks good, just a couple minor changes.

**Review by realshuting — APPROVED** [review]

**Pull request merged**